Tests/test_content.py

```diff
@@ -43,8 +40,9 @@
 from Tests.test_integration import Docker, check_integration, disable_all_integrations
 from Tests.test_dependencies import get_used_integrations, get_tests_allocation_for_threads
 from demisto_sdk.commands.common.constants import FILTER_CONF, PB_Status
-from demisto_sdk.commands.common.tools import print_color, print_error, print_warning, \
-    LOG_COLORS, str2bool
+from demisto_sdk.commands.common.tools import str2bool
+
+logging_manager: ParallelLoggingManager = None
```

Comment:
when are we using logging module and when are we using logging manager?

Reply:
There are common methods that are being used from other steps as well.
These steps have logging module installed only.

This way those steps will still print in real time while the test content step will send the logs to the logs manager to handle